third_party/libsdl2/tnt/CMakeLists.txt

```diff
@@ -163,7 +162,12 @@ target_compile_options(${TARGET}
 target_include_directories(${TARGET} PUBLIC ${PUBLIC_HDR_DIR})
 
 if (WIN32)
-    target_compile_options(${TARGET} PRIVATE -Wno-pragma-pack)
+    # Making this option public because the warning occurs in public headers
+    target_compile_options(${TARGET} PUBLIC -Wno-pragma-pack)
+    target_compile_options(${TARGET} PRIVATE -Wno-unused-local-typedef)
     # Windows needs a WinMain static library.
     add_library(${TARGET}main ${SRC_DIR}/main/windows/SDL_windows_main.c)
-endif ()
+    target_compile_options(${TARGET}main PRIVATE -Wno-pragma-pack)
```

Comment:
`${TARGET}main`? Is this a typo? Why is `-Wno-pragma-pack` listed twice?

Reply:
In this case it's not a typo. ${TARGET}main and ${TARGET} are separate targets.

Reply:
Oops sorry I missed it a few lines above